@cgsavard 

There is an unused variable that is causing the build errors:
```
L1Trigger/L1TTrackMatch/plugins/TPFastJetProducer.cc
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_14_0_X_2023-11-21-2300/src/L1Trigger/L1TTrackMatch/plugins/TPFastJetProducer.cc:146:9: warning: variable 'nStubLayerTP_g' set but not used [-Wunused-but-set-variable]
```

--- a/L1Trigger/L1TTrackMatch/plugins/TPFastJetProducer.cc
+++ b/L1Trigger/L1TTrackMatch/plugins/TPFastJetProducer.cc
@@ -143,12 +143,9 @@ void TPFastJetProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetu
     }
 
     int nStubLayerTP = 0;
-    int nStubLayerTP_g = 0;
     for (int isum : hasStubInLayer) {
       if (isum >= 1)
         nStubLayerTP += 1;
-      if (isum == 2)
-        nStubLayerTP_g += 1;
     }
 
     // tp quality cuts to match L1 tracks

